runtime/ms-rest-azure/lib/cloudError.js

```diff
@@ -95,11 +101,19 @@ class CloudError extends Error {
                 }
               }
             }
+          },
+          innerError: {
+            required: false,
+            serializedName: 'innerError',
```

Comment:
The [odata v4.0 spec](http://docs.oasis-open.org/odata/odata-json-format/v4.0/os/odata-json-format-v4.0-os.html#_Toc372793091) and the [Azure Resource Manager RPC spec](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content) says that the name of the property is `innererror`.

> The value for the innererror name/value pair MUST be an object. The contents of this object are service-defined. Usually this object contains information that will help debug the service. The innererror name/value pair SHOULD only be used in development environments in order to guard against potential security concerns around information disclosure.

The PR states the name of the property is innerError (camelCase).

Reply:
You are right. Changed the name to innererror